src/lib/server/websearch/parseWeb.ts

```diff
@@ -32,7 +32,9 @@ export async function parseWeb(url: string) {
 		r.headers.get("content-type")?.includes("text/plain") ||
 		r.headers.get("content-type")?.includes("text/markdown")
 	) {
-		return r.text();
+		const text = await r.text();
+		// JSON.stringify is needed to turn string concatenation into a single string (ex: "Hello, " + "world!" -> "Hello, world!")
```

Comment:
```shell
> JSON.stringify('"Hello, " + "world!"')
'"\\"Hello, \\" + \\"world!\\""'
```

It doesn't give `"Hello, world!"`

Not sure I understand where the concat happens

Reply:
example:
```ts
const text = await r.text();
console.log({text})
// {text: "Hello " + "world"}
```

and TEI endpoint was failing with `Invalid json`

```ts
let text = r.text();
text = JSON.stringify(text)
console.log({text})
// {text: "Hello  world"}
```

TEI endpoint works fine

Reply:
will let @nsarrazin review I don't understand it